bestfirstsearch.java

```diff
@@ -134,12 +142,12 @@ public void bestfirstsearch()
 System.out.print(arr[i]+"\t");		//stores close list
 }
 System.out.println("");
-int o=arr.length-1;
+@Positive int o=arr.length-1;
 while(arr[o]==0){
 o--;
 }
 int arr1[]=new int[v];
-int r=0;
+@Positive int r=0;
```

Comment:
This is a contradiction. Please read the manual section of `@Positive`.

Reply:
Sorry for this...I have refereed manual and removed this